packages/core/src/components/AlertDialogDelete.tsx

```diff
@@ -36,7 +44,7 @@ export function AlertDialogDelete({
       onClose={onClose}
     >
       <AlertDialogOverlay>
-        <AlertDialogContent>
+        <AlertDialogContent bg={backgroundColor}>
```

Comment:
I'm not sure if this is the right way to do it. This way we'll need to add a prop to every element we support and then drill this value to the element. Can we define the themes and the **current theme** in a top level context or zustand file somewhere and have that imported in the actual elements. This way we don't need to keep on passing the color. The actual component can refer this zustand store to identify what the currently chosen theme is and style it accordingly.